x-pack/solutions/security/plugins/security_solution/public/app/solution_navigation/navigation_tree.test.ts

```diff
@@ -0,0 +1,1102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SolutionLinkCategory, SolutionNavLink } from '../../common/links';
+import { formatNavigationTree } from './navigation_tree';
+
+describe('formatNavigationTree', () => {
```

Comment:
This is the first PR for https://github.com/elastic/kibana/issues/212903. The snapshots in this unit test are a foundation for the future PRs, which will bring small modifications to the snapshots.